File: PRAC/W7/sort_words.pl
#!/usr/bin/perl

while ($line = <STDIN>) {
	@array = split(' ', $line);
	@sorted = ();
	foreach $c (sort(@array)) {
		push(@sorted, $c);
	}
	foreach $item (@sorted) {
		print $item." ";
	}
	print "\n";
}

File: PRAC/W9/sort_words.py
#!/usr/bin/python

import sys
import re

for line in sys.stdin:
	array = []
	line = line.replace("\n", "")
	array = line.split(" ")
	array.sort()
	string = ""
	for i in array:
		if i == "":
			continue;
		#print("ITEM = {}".format(i))
		string += i + " "
	print(string)
